app/views/api/v0/organizations/show.json.jbuilder

```diff
@@ -0,0 +1,18 @@
+json.type_of "organization"
+
+json.extract!(
+  @organization,
+  :username,
+  :name,
+  :summary,
+  :twitter_username,
+  :github_username,
+  :url,
+  :location,
+  :tech_stack,
+  :tag_line,
+  :story,
+)
+
+json.joined_at @organization.created_at.strftime("%b %e, %Y")
```

Comment:
This as all the other dates should be in ISO8601/RFC3339 format as specified by the spec. Unfortunately `joined_at` being a "human date" in the user object is a remnant of the past.

Reply:
Thanks @rhymes for reviewing. I was following the user show endpoint but just uploaded a change to the iso format.